static/app/views/performance/transactionSummary/utils.tsx

```diff
@@ -149,9 +149,14 @@ export function generateReplayLink(routes: PlainRoute<any>[]) {
     const replaySlug = `${tableRow['project.name']}:${replayId}`;
     const referrer = encodeURIComponent(getRouteStringFromRoutes(routes));
 
+    const transactionTimestamp = tableRow.timestamp
+      ? Math.floor(new Date(tableRow.timestamp).getTime() / 1000) * 1000
```

Comment:
is the `new Date` needed? Looks like your turning a timestamp into a date into a timestamp again

Reply:
Yup! I do this to change the timestamp format